UTGame/Src/SpectatorUI/Classes/SpectatorUI_Interaction.uc

```diff
@@ -118,8 +118,31 @@ function Cleanup() {
 }
 
 function bool ShouldRender() {
-    // the same condition appears in UTHUD::DrawGameHud
-    return PlayerReplicationInfo != None && (PlayerReplicationInfo.bOnlySpectator || Outer.IsInState('Spectating'));
+	local GameReplicationInfo GRI;
+	local UTGameReplicationInfo UTGRI;
+	local UTHUD HUD;
+	
+	// the same condition appears in UTHUD::DrawGameHud
+	if (PlayerReplicationInfo == none || !PlayerReplicationInfo.bOnlySpectator || !Outer.IsInState('Spectating'))
```

Comment:
That's not the negation of the original condition. It should be `PlayerReplicationInfo == none || !PlayerReplicationInfo.bOnlySpectator && !Outer.IsInState('Spectating')` instead, no?